packages/credentials/package.json

```
@@ -85,7 +85,7 @@
     "@typescript-eslint/eslint-plugin": "6.4.0",
     "@typescript-eslint/parser": "6.4.0",
     "@web5/common": "0.2.1",
-    "@web5/crypto": "0.2.2",
+    "@web5/crypto": "0.2.3",
```

`@web5/credentials` needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```